renderer/main.js

```diff
@@ -109,7 +109,7 @@ function onState (err, _state) {
 
   // Log uncaught JS errors
   window.addEventListener('error',
-    (e) => telemetry.logUncaughtError('window', e.error), true)
+    (e) => telemetry.logUncaughtError('window', e.error || e.target), true)
```

Comment:
When I play an unsupported video (xvid) I get this exception in the console:
```
Uncaught (in promise) DOMException: Failed to load because no supported source was found.
```
The file and line is reported as `undefined:1`, I don't know if this is a bug in chromium?
No matter how I tried to catch this, the event never had the `error` property, so i added target (the `<video>` element) to at least provide some info about where the error happened, because the error message from the console is not saved anywhere.

We might be able to solve this in another way since I also get the _"Sorry, we can't play that file"_ modal instead of _"Video codec unsupported"_.

I will try to look into this.